### PR TITLE
Fix moderation admin

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

**Comment by johngian** [comment]
@akatsoulas r?

File: mresponse/moderations/admin.py [hunk 1]
```diff
@@ -6,3 +6,26 @@
 @admin.register(moderations_models.Moderation)
 class ModerationAdmin(admin.ModelAdmin):
     readonly_fields = ['submitted_at']
+    list_display = (
+        'pk',
```
Comment (akatsoulas) [inline mresponse/moderations/admin.py]: Just flagging this to make sure it's not a mistake and you need the `pk` in the list display.

File: mresponse/moderations/admin.py [hunk 2]
```diff
@@ -6,3 +6,26 @@
...
+        'positive_in_tone',
+        'addressing_the_issue',
+        'personal',
+        'karma_points',
+        'submitted_at'
+    )
+
+    def get_moderator(self, obj):
+        if obj.moderator.profile:
+            if obj.moderator.profile.name:
```
Comment (akatsoulas) [inline mresponse/moderations/admin.py]: Can a profile be without a name?

Comment (akatsoulas) [reply mresponse/moderations/admin.py]: Also you can combine this with an `and` statement to make it a bit simpler to read

**Pull request merged**